### ISSUE TITLE
Form Validation - Defect

### ISSUE THREAD

**Issue by nbaggett** [erozhon/CSC308-Rideshare#75]
**Description:** Currently can enter a negative price for a ride and no limit on the seating capacity. When this is integrated into the front-end be sure to prevent the user from putting a negative price or from putting a seating capacity in their car of more than 10. 

**Reproduction steps:** Simply enter a negative number for the price or a very large number for the capacity.

**Expected behavior:** Prevent the user from entering bad values for these things.

**Observed behavior:** Was able to enter bad values.

It is worth noting that I tested the backend entry of the data which shouldn't have validation. These values should be tested on the frontend before form submittal, whenever the back-end is integrated don't forget to validate these :)

**Comment by bcdasilv** [comment]
@nbaggett you're correct with the defect, however, data validation should also be done at the backend layer.  Actually, if you have to choose, backend data validation should be prioritized. Remember that, once your backend is decoupled from your frontend through a REST API, you can't always guarantee that the data will come validated from calls coming from frontend components. Your backend is supposed to validate data before proceeding with operations that need data validation. And depending on the feature the data is involved, this might incur in security issues.
